services/movie.py

```diff
@@ -0,0 +1,41 @@
+from typing import Optional
+
+from django.db.models import QuerySet
+
+from db.models import Genre, Actor, Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet[Movie]:
+    if not genres_ids and not actors_ids:
+        return Movie.objects.all()
+    if genres_ids and actors_ids:
+        return Movie.objects.filter(genres__in=genres_ids,
+                                    actors__in=actors_ids).distinct()
```

Comment:
The query in `get_movies` when both `genres_ids` and `actors_ids` are provided may not work as expected. It uses an `AND` condition between genres and actors, which might not return any results if a movie does not have both the specified genres and actors. Consider whether this logic aligns with the intended functionality.